sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -11,15 +11,12 @@ import com.mapbox.maps.renderer.gl.GlUtils.toFloatBuffer
 internal class BitmapWidgetRenderer(
   @Volatile
   private var bitmap: Bitmap?,
-  private val position: WidgetPosition,
-  private val marginX: Float,
-  private val marginY: Float,
+  @Volatile
+  private var widgetPosition: WidgetPosition
```

Comment:
Could we just expose the property `position` as `internal var position` and override the `set`?

Reply:
Do we really need `@Votalite` here?

Reply:
Updated, please take another look.
I added `@Votalite` because we might read and set the `widgetPosition` in different threads, we want to always to draw using the updated position, similarly to the bitmap argument above, or do you have some other suggestions?

Reply:
Clarified offline, we can follow up later.